UnitTests/BraintreePayPalTests/BTPayPalRequest_Tests.swift

```diff
@@ -50,7 +50,7 @@ class BTPayPalRequest_Tests: XCTestCase {
 
         request.lineItems = [BTPayPalLineItem(quantity: "1", unitAmount: "1", name: "item", kind: .credit)]
 
-        let parameters = request.parameters(with: configuration)
+        let parameters = request.baseParameters(with: configuration)
```

Comment:
🧶 : Any reason for the method name change here?

Reply:
Mainly for clarity. We used `parameters(with: )` in `BTPayPalRequest` as well as `BTPayPalCheckoutRequest` and `BTPayPalVaultRequest`. But in `BTPayPalCheckoutRequest` and `BTPayPalVaultRequest` we were then adding to the parameters from `BTPayPalRequest`. Made more sense to clarify what each function was doing since they are just used internally and `BTPayPalRequest` is only responsible for constructing the `baseParameters` then `BTPayPalCheckoutRequest` and `BTPayPalVaultRequest` add to those parameters to create the parameters for the request.